tests/plugins/test_minimap.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+
+"""
+Test FloatImage
```

Comment:
Wrong title. Also a title is not necessary IMO, it's the same as the filename.

Reply:
fixed